simcon_project/simcon_project/urls.py

```diff
@@ -15,14 +15,12 @@
 """
 from django.contrib import admin
 from django.urls import path
-from users.views import RedirectFromLogin, StudentView, ResearcherView, ViewResponse, 
-  UpdateOverallResponseFeedback, UpdateNodeResponseFeedback, SettingsView, TemplateStartView
+from users.views import RedirectFromLogin, StudentView, ResearcherView, ViewResponse, \
```

Comment:
whats the `\` at the end of the line?

Reply:
It's a line break in Python. It's needed since the line goes past 140 characters or something and you get a yellow squiggly otherwise.